src/Commands/InitCommand.php

```diff
@@ -52,6 +52,12 @@ class InitCommand extends Command implements Isolatable
 
     protected string $readmeContent = '';
 
+    protected array $composerCommands = [
+        'composer require ronasit/laravel-helpers',
+        'composer require --dev ronasit/laravel-swagger',
```

Comment:
@DenTray, should you also change it in packagist? - 
![image](https://github.com/user-attachments/assets/3f8d8010-6677-4b26-ac64-a0f818d29119)

I receive this message - The package you required is recommended to be placed in require-dev (because it is tagged as "testing") but you did not use --dev.
Do you want to re-run the command with --dev? [yes]?

Reply:
@pirs1337 yep, let's change it for packagist as well